### PR TITLE
changed background color of <add new project> button and added plus icon before it

### PR DESCRIPTION
**Pull request by Sarvesh-Ghildiyal**
Targets #{ISSUE_ID}
2314

### Description
I changed background color of <add new project> button and added plus icon before it

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by Ayush8923** [comment]
@Sarvesh-Ghildiyal Please enter the issue link in your PR. And also resolve the merge conflict.

**Comment by gautam071** [comment]
@Sarvesh-Ghildiyal Can you resolve the merge conflicts?

File: Modules/Project/Resources/views/index.blade.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
         @include('project::menu_header')
         @can('projects.create')
             <span class='mt-4'>
-                <a href= "{{ route('project.create') }}" class="btn btn-success text-white"> <i class="fa fa-plus mr-1"></i>{{ __('Add new project') }}</a>
+                <a href= "{{ route('project.create') }}" class="btn btn-success text-white"><i class="fa fa-plus" style="font-size:14px"></i> {{ __('Add new project') }}</a>
```
Comment (Ayush8923) [inline Modules/Project/Resources/views/index.blade.php:16]: Please use bootstrap class instead of normal CSS.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comments.

File: resources/views/knowledgecafe/library/books/index.blade.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
         </div>
         @can('library_books.create')
             <div class="col-6">
-                <a href="{{ route('books.create') }}" class="btn btn-success float-right"><i class="fa fa-plus mr-1"></i> Add New Book</a>
+                <a href="{{ route('books.create') }}" class="btn btn-success float-right"><i class="fa fa-plus fs-1 mr-12"></i> Add New Book</a>
```
Comment (Ayush8923) [inline resources/views/knowledgecafe/library/books/index.blade.php:16]: Why have you made changes to this file? While there was no need to make changes to this file.

File: Modules/Project/Resources/views/index.blade.php [hunk 3]
```diff
@@ -13,7 +13,7 @@
         @include('project::menu_header')
         @can('projects.create')
             <span class='mt-4'>
-                <a href= "{{ route('project.create') }}" class="btn btn-success text-white"> <i class="fa fa-plus mr-1"></i>{{ __('Add new project') }}</a>
+                <a href= "{{ route('project.create') }}" class="btn btn-success text-white"><i class="fa fa-plus fs-5"></i> {{ __('Add new project') }}</a>
```
Comment (Ayush8923) [inline Modules/Project/Resources/views/index.blade.php:16]: Why did you add the `fs-5` class?

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comments.

**Review by gautam071 — APPROVED** [review]
Looks good!

@Ayush8923 Can you take a look at this PR once?

**Review by Ayush8923 — APPROVED** [review]
LGTM.

**Pull request merged**